src/Monitoring/MetricId.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * @author    Craig Gosman <craig@ingenerator.com>
+ * @licence   proprietary
```

Comment:
Couple of `@licence` tags

Reply:
license tags are wrong on every single file in this project. I could fix them all in a single commit at the end of this or put it in another pull entirely.